app/views/musicians/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="banner" style="background-image: linear-gradient(-225deg, rgba(0,101,168,0.6) 0%, rgba(0,36,61,0.6) 50%), url('background.jpg');">
+  <div class="banner-content">
+    <h1>Some Stuff</h1>
+    <p>Very cool stuff, useful and smart</p>
+    <a class="btn btn-primary btn-lg">GET IT</a>
+  </div>
+</div>
+
+
+
+
+
+<% @musicians.each do |musician| %>
+  <% @musicians.each do |musician| %>
+  <%= link_to musician.name, musician_path(musican) %>
+<% end %>
+
+<a href="/musicians/1">Paul</a>
+<a href="/musicians/2">Ringo</a>
+<a href="/musicians/3">Jimmy</a>
```

Comment:
Hard coded links like these are a `no go`.

Rails comes with a `link_to` helper for this.